assets/src/application/services/context/RelationsProvider.tsx

```diff
@@ -1,17 +1,13 @@
-/**
- * External imports
- */
 import React, { createContext } from 'react';
 
 import useRelationsManager from '../apollo/relations/useRelationsManager';
 import { RelationsManager } from '../apollo/relations';
-import { ProviderProps } from './types';
 
 const RelationsContext = createContext<RelationsManager | null>(null);
 
 const { Provider, Consumer: RelationsConsumer } = RelationsContext;
 
-const RelationsProvider: React.FC<ProviderProps> = ({ children }) => {
+const RelationsProvider: React.FC = ({ children }) => {
 	const relations = useRelationsManager();
 	return <Provider value={relations}>{children}</Provider>;
 };
```

Comment:
I have simplified the types for context providers

Reply:
nice, this change adds more visibility :+1: